Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM python:3.8-slim
+ENV PYTHONUNBUFFERED=1
 WORKDIR /mote
 COPY requirements.txt requirements.txt
 RUN pip install -r requirements.txt
 RUN sed -i 's|json_cache_location = "/var/cache/httpd/mote/cache.json"|json_cache_location = "./cache.json"|' ./mote/config.py
-EXPOSE 5000
 COPY . .
-CMD ["python3", "runmote.py"]
+EXPOSE 5000
+CMD ["python3", "runmote.py"]
```

Comment:
We wouldn't need it once we have the like in Docker-compose: https://github.com/fedora-infra/mote/pull/150/commits/7de8aa6f21528aff693bb49545dae274fc4f74e9#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R5

Reply:
A note to myself `REMOVE L#9 of Dockerfile`